Formula/p/prjtrellis.rb

```diff
@@ -0,0 +1,42 @@
+class Prjtrellis < Formula
+  desc "Documenting the Lattice ECP5 bit-stream format"
+  homepage "https://github.com/SymbiFlow/prjtrellis"
+  url "https://github.com/YosysHQ/prjtrellis.git",
+      tag:      "1.4",
+      revision: "f1e5710099313d2e1862d4ef2582293f6b7ee122"
```

Comment:
also try to use `https://github.com/YosysHQ/prjtrellis/archive/refs/tags/1.4.tar.gz` and use `https://github.com/YosysHQ/prjtrellis/releases/download/1.4/prjtrellis-db-1.4.zip` for `database` via buildpath setup